.test-infra/kubernetes/kafka-cluster/05-kafka/configmap-config.yaml

```diff
@@ -34,5 +34,5 @@ data:
       sleep 20
     done
     echo "Applying runtime configuration using confluentinc/cp-kafka:5.0.1"
-    kafka-topics --zookeeper zookeeper:2181 --create --if-not-exists --force --topic apache-beam-load-test --partitions 3 --replication-factor 2
+    kafka-topics --zookeeper zookeeper:2181 --create --if-not-exists --force --topic apache-beam-load-test --partitions 1 --replication-factor 1
```

Comment:
Was # of partitions also affecting the resulting hash? If not, maybe it makes sense to have more than one partition? Otherwise the data is just being read serially, right?

Reply:
The increase of number of partitions introduces a possibility that not all messages will be read from the topic and the test will hang. Keeping it at 1 ensures that the tests finish